src/pages/home/report/FloatingMessageCounter/index.js

```diff
@@ -11,14 +11,8 @@ import withLocalize, {withLocalizePropTypes} from '../../../../components/withLo
 import FloatingMessageCounterContainer from './FloatingMessageCounterContainer';
 
 const propTypes = {
-    /** Count of new messages to show in the badge */
-    count: PropTypes.number,
-
     /** Whether the marker is active */
```

Comment:
```suggestion
    /** Whether the New Messages indicator is active */
```
Should we rename this file since it's no longer showing counts?